test/core/util/fuzzer_corpus_test.cc

```diff
@@ -70,12 +70,14 @@ class ExampleGenerator
     if (examples_.empty()) {
       if (!FLAGS_file.empty()) examples_.push_back(FLAGS_file);
       if (!FLAGS_directory.empty()) {
+#ifndef BAZEL_BUILD
```

Comment:
this seems a bit too adhoc and "BAZEL_BUILD" doesn't seem to comply with the define names normally used with gRPC C and C++ code?

Reply:
@vjpai  you know more about this than I do, can you please review?